### PR TITLE
Added fullstops and colons for consistency.

### PR DESCRIPTION
**Pull request by GitHJet**

### CODE REVIEW

File: docs/source/setup.md [hunk 1]
```diff
@@ -55,7 +55,7 @@ Don't forget to replace YOUR_GITHUB_USER_NAME with your git username.
 
 ### Step 3: Setup code base
 
-* Create a python virtual environment and install python dependencies.
+* Create a python virtual environment and install python dependencies
```
Comment (RishabhJain2018) [inline docs/source/setup.md]: Please don't remove full-stop from the end. I think the earlier version looks better.

File: docs/source/setup.md [hunk 2]
```diff
@@ -97,7 +97,7 @@ source venv/bin/activate
 python manage.py runserver --settings=settings.dev
 ```
 
-* To run frontend development server for at [http://127.0.0.1:8888](http://127.0.0.1:8888)
+* To run front-end development server for at [http://127.0.0.1:8888](http://127.0.0.1:8888).
```
Comment (RishabhJain2018) [inline docs/source/setup.md]: Please check as I think there is some grammatical mistake in this line.

File: docs/source/setup.md [hunk 3]
```diff
@@ -202,15 +202,15 @@ npm install -g bower
 
 ### Step 4: Start the development environment
 
-* To run backend development server at [backend]
+* To run backend development server at [backend]:
```
Comment (RishabhJain2018) [inline docs/source/setup.md]: Please specify the port number as `127.0.0.1:8000` here as done in above command.

File: docs/source/setup.md [hunk 4]
```diff
@@ -202,15 +202,15 @@ npm install -g bower
...
 
 ```
 # activate virtual environment if not activated
 source venv/bin/activate
 python manage.py runserver --settings=settings.dev
 ```
 
-* To run frontend development server for at [frontend]
+* To run frontend development server for at [frontend]:
```
Comment (RishabhJain2018) [inline docs/source/setup.md]: Please specify the port number as `127.0.0.1:8888` here as done in above command.

**Comment by gautamjajoo** [comment]
@GitHJet Hey! are you working on this issue or can I complete it and close the PR? :)

File: docs/source/setup.md (hunk 1 above)
Comment (GitHJet) [reply docs/source/setup.md]: Have removed the full-stops.

Comment (RishabhJain2018) [reply docs/source/setup.md]: Please revert to earlier version i.e `dependencies.` in this line only.

**Comment by RishabhJain2018** [comment]
@gautamjajoo I think @GitHJet is working on it. Please try to take up any other issue from the list.

**Comment by RishabhJain2018** [comment]
@GitHJet Any update on this ?

**Comment by GitHJet** [comment]
Apologise for the late edits. Still learning to use GitHub.

**Comment by aka-jain** [comment]
@RishabhJain2018 @deshraj please have a look here.

**Comment by deshraj** [comment]
@GitHJet can you please make the requires changes ASAP?

**Comment by GitHJet** [comment]
@deshraj I thought I have made the changes already. I'm not sure what else there needs to be done.

File: docs/source/setup.md [hunk 5]
```diff
@@ -202,15 +203,15 @@ npm install -g bower
 
 ### Step 4: Start the development environment
 
-* To run backend development server at [backend]
+* To run backend development server at [127.0.0.1:8000]:
```
Comment (RishabhJain2018) [inline docs/source/setup.md]: Please change `[127.0.0.1:8000]` -> `[http://127.0.0.1:8000](http://127.0.0.1:8000)`

File: docs/source/setup.md [hunk 6]
```diff
@@ -202,15 +203,15 @@ npm install -g bower
...
 
 ```
 # activate virtual environment if not activated
 source venv/bin/activate
 python manage.py runserver --settings=settings.dev
 ```
 
-* To run frontend development server for at [frontend]
+* To run frontend development server at [127.0.0.1:8888]:
```
Comment (RishabhJain2018) [inline docs/source/setup.md]: Please change `[127.0.0.1:8888]` -> `[http://127.0.0.1:8888](http://127.0.0.1:8888)`

**Comment by spyshiv** [comment]
@RishabhJain2018 Assigning this to you. Please do the required changes and merge this PR.

**Comment by deshraj** [comment]
The changes are not significant and hence closing it.

**Pull request closed without merging**